frontend/server/src/DAO/Groups.php

```diff
@@ -48,8 +48,15 @@ public static function getByName(string $name) : ?\OmegaUp\DAO\VO\Groups {
 
     /**
      * Returns all groups that a user can manage.
+     * @param int $userId
+     * @param int $identityId
+     * @return array<int, array<string, mixed>>
      */
-    final public static function getAllGroupsAdminedByUser($user_id, $identity_id) {
+    final public static function getAllGroupsAdminedByUser(
+        int $userId,
+        int $identityId
+    ) : ?array {
```

Comment:
```suggestion
    ) : array {
```

aquí el `?` no es necesario porque jamás regresas `null`.

Reply:
Listo